modules/about-oadp-update-channels.adoc

```diff
@@ -9,23 +9,25 @@
 
 When you install an OADP Operator, you choose an _update channel_. This channel determines which upgrades to the OADP Operator and to Velero you receive. You can switch channels at any time.
 
-There are three update channels:
+The available update channels:
 
-* The *stable* channel contains the latest minor updates (y-stream updates) and patches (z-stream updates) of OADP ClusterServiceVersion`. As each new release is published, the available `ClusterServiceVersion` of the OADP Operator will be appended with the latest available minor patch.
+* The *stable* channel is now *deprecated*. The *stable* channel contains the patches (z-stream updates) of OADP `ClusterServiceVersion` for `oadp.v1.1.z` and older versions from `oadp.v1.0.z`.
 
 * The *stable-1.0* channel contains `oadp.v1.0._z_`, the most recent OADP 1.0 `ClusterServiceVersion`.
 
 * The *stable-1.1* channel contains `oadp.v1.1._z_`, the most recent OADP 1.1 `ClusterServiceVersion`.
 
+* The *stable-1.2* channel contains `oadp.v1.2._z_`, the most recent OADP 1.2 `ClusterServiceVersion`.
+
 *Which update channel is right for you?*
 
-* Choose the *stable* update channel to install the latest stable OADP version and to receive both minor updates and patches. If you choose this channel, you will receive all y-stream and all z-stream updates for version _x.y.z_.
+* The *stable* channel is now deprecated.  If you are already using the stable channel you will continue to get updates from `oadp.v1.1._z_`.
 
```

Comment:
I'd format stable in "stable channel" (second sentence) as you did in the first sentence. Up to you.